recipes-kernel/kernel-tests/kernel-performance-tests-files/run-cyclictest

```diff
@@ -58,6 +58,7 @@ function run_cyclictest()
     LOG="$LOG_DIR/cyclictest-$1-`date +'%Y_%m_%d-%H_%M_%S'`.log"
     cyclictest --smp --priority=98 --mlockall --interval=997 --quiet --duration="$TEST_DURATION" --histofall=1000 --histfile="$LOG" > /dev/null
     add_system_info "$LOG"
+    python3 upload_cyclictest_results.py "-i $LOG" "-s dso-db-influx-1.ni.systems" "-p 8086"
```

Comment:
What happens when someone runs this test from external? Does it fail gracefully?

Reply:
The test still passes but you get an error print from the upload script:
```
# ptest-runner 
START: ptest-runner
2022-10-14T17:12
BEGIN: /usr/lib/kernel-performance-tests/ptest
ERROR:root:influxdb error: HTTPConnectionPool(host='unknown.server.foo', port=8086): Max retries exceeded with url: /write?db=rtos_kernel_performance (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc47d4bf2b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
cyclictest with idle load latency: 40 (usec) is less than max latency: 300 (usec)
histogram log file: /var/local/ptest-results/kernel-performance-tests/cyclictest-idle-2022_10_14-17_12_27.log
PASS: test_kernel_cyclictest_idle
END: /usr/lib/kernel-performance-tests/ptest
2022-10-14T17:12
STOP: ptest-runner
```

I can turn the error into a warning if that's preferable.

Reply:
Yeah; let's make it a warning. Then I'll be fine with it.